drivers/storage/portworx/component/telemetry.go

```diff
@@ -109,6 +110,9 @@ const (
 	defaultCollectorPort    = 10000
 	defaultRegisterPort     = 12001
 	defaultPhonehomePort    = 12002
+
+	arcusPingInterval = 6 * time.Second
```

Comment:
Do we need 30 secs in total?

Reply:
yes that's the decision after discussion.